docs/cellxgene_census_FAQ.md

```diff
@@ -0,0 +1,109 @@
+
+- Open and read data at low latency from the cloud.
+- Query and access data using metadata filters.
+- Load and create AnnData objects.
+- Load and create Seurat objects.
+- From Python create PyArrow objects, SciPy sparse matrices, NumPy arrays, and pandas data frames.
+- From R create R Arrow objects, sparse matrices (via the Matrix package), and standard data frames and (dense) matrices.
+
+## Can I query human and mouse data in a single query?
+
+At the moment it is not possible to query both mouse and human data in one query. This is due to the data from these organisms using different [organism-specific gene annotations](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md#required-gene-annotations).
+
+## Where are the Census data hosted?
+
+The Census data is publicly hosted free-of-cost in an Amazon Web Services (AWS) S3 bucket in the [`us-west-2` region](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions).
+
+## How can I increase the performance of my queries?
+
+Since the access patterns are via the internet, usually the main limiting step for data queries is bandwidth and client location.
+
+We recommend the following to increase query efficency: 
+
+- Utilize a computer connected to high-speed internet.
+- Utilize an ethernet connection and not a wifi connection.
+- If possible utilize online computing located in the west coast of the US.
+   - Highly recommended: [EC2 AWS instances](https://aws.amazon.com/ec2/) in the `us-west-2` region.
+
+## Can I use conda to install the Census Python API?
+
+Currently there is not conda package available for `cellxgene-census`. However you can use conda in combination with `pip` to install the package in a conda environment:
+
+```bash
+conda create -n census_env python=3.10
+conda activate census_env
+pip install cellxgene-census
+```
+
+## How can I ask for support?
+
+You can either submit a [github issue](https://github.com/chanzuckerberg/cellxgene-census/issues/new/choose) or post in the slack channel `#cellxgene-census-users` at the [CZI Slack community](https://cziscience.slack.com/join/shared_invite/zt-czl1kp2v-sgGpY4RxO3bPYmFg2XlbZA#/shared-invite/email).
+
+## How can I ask for new features?
+
+You can submit a [feature request in the github repository](https://github.com/chanzuckerberg/cellxgene-census/issues/new?assignees=&labels=user+request&template=feature-request.md&title=).
+
+## How can I contribute my data to the Census?
+
+You first need to submit your data to [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/), then it will automatically get included in the Census if it meets the [biological criteria defined in the Census schema](https://github.com/chanzuckerberg/cellxgene-census/blob/main/docs/cellxgene_census_schema.md#data-included). 
```

Comment:
"You first need to submit" sounds like everyone can submit which may not be true in the future. 

_We confirm that we will accept your data._